app/globals.css

```
@@ -170,21 +170,32 @@ a {
+        color: var(--primary-hover) !important;
+        background-color: var(--accent);
+        border-radius: 0.25rem;
+        text-decoration-color: var(--primary-hover) !important;
     }
+
 }
 
+.nav-item a, a.nav-item {
+    color: var(--primary) !important;
+    text-decoration-color: var(--primary) !important;
+    text-decoration: none !important;
```

Biome (2.1.2)

[error] Unexpected shorthand property text-decoration after text-decoration-color



(lint/suspicious/noShorthandPropertyOverrides)